### ISSUE TITLE
Widget help function no longer working

### ISSUE THREAD

**Issue by wvdvegte** [biolab/orange3#6609]
**What's wrong?**
This is probably either since I updated to Orange 3.36.1 for Mac, or to Mac OS Sonoma: when clicking on the question mark in the bottom left of any widget's dialog box, nothing happens.

**How can we reproduce the problem?** 
Click on the "?" on the bottom left of any widget dialog box, on a system with specifications similar to mine.

**What's your environment?**

- Operating system: Mac OS Sonoma (14.0)
- Orange version: 3.36.1 (Apple ARM)
- How you installed Orange: From DMG

**Comment by ales-erjavec** [comment]
This should be fixed in https://github.com/biolab/orange-canvas-core/pull/281.

Can you try running 
```sh
/Applications/Orange.app/Contents/MacOS/pip install orange-canvas-core==0.1.34
```
in the Terminal.app

**Comment by wvdvegte** [comment]
@ales-erjavec, I tried this but I got an error message:

` ERROR: Failed to restore /Applications/Orange.app/Contents/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/orange_canvas_core-0.1.33.dist-info/
  ERROR: Failed to restore /Applications/Orange.app/Contents/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/orangecanvas/
ERROR: Could not install packages due to an OSError: [Errno 1] Operation not permitted: '/Applications/Orange.app/Contents/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/orangecanvas'`

I can't run Orange - I'm, going to reinstall it.

**Comment by wvdvegte** [comment]
I have now updated to Orange 3.36.2 but the ? button is still unresponsive

**Comment by wvdvegte** [comment]
I just learned from my students they have this problem in Windows, too.

**Comment by PrimozGodec** [comment]
I just tested the issue and can confirm that the error should be fixed in https://github.com/biolab/orange-canvas-core/pull/281. This can be fixed by installing a new installer with version 3.36.2. It was already published on the website (a week ago) but was removed this morning since we noticed one other issue. We are building a new installer and reuploading it to the website soon. 

Even though this issue is already solved, I am keeping it open until we upload a new installer. I'll post it when it's ready.

**Comment by PrimozGodec** [comment]
We just published new 3.36.2 packages that should fix this problem at https://orangedatamining.com/download/
Please reinstall Orange. :)

**Comment by wvdvegte** [comment]
@PrimozGodec, thanks for resolving this. However, I'm not sure how best to reinstall Orange. I had some problems recently with installing add-ons, which I may have resolved by installing the Intel version of 3.36.1 on my Silicon Mac, and updating from there. Will the 3.36.2 for Mac [on the download page](https://orangedatamining.com/download/) allow me to reinstall all the add-ons without problems?

**Comment by wvdvegte** [comment]
Thanks, it's working fine now.
